docs/content/guides/dagster/non-argument-deps.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Assets without Arguments and Return Values | Dagster
+description: Often we don't want to build or load our assets as Python objects in memory
+---
+
+# Assets without arguments and return values
```

Comment:
The content here is written with the assumption that readers are coming from the immediately-prior section of the tutorial. E.g. the first sentence references nabisco_cereals, which was defined earlier in the tutorial.

I think it needs to be amended to make sure it's decipherable without that context.